src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -387,6 +392,62 @@ public FeatureDataSource( final File featureFile, final FeatureCodec<T, ?> codec
     }
 
     /**
+     * Returns the sequence dictionary created from the index or null if that fails.
```

Comment:
Comment is inaccurate -- should be `Returns the sequence dictionary for this source of Features. Uses the dictionary from the VCF header (if present) for variant inputs, otherwise attempts to create a sequence dictionary from the index file (if present). Returns null if no dictionary could be created from either the header or the index.`